content/en/tracing/trace_collection/library_config/java.md

```diff
@@ -220,6 +220,18 @@ When set to `true` query string parameters and fragment get added to web server
 **Default**: `true`<br>
 When set to `false` http framework routes are not used for resource names. _This can change resource names and derived metrics if changed._
 
+`dd.trace.http.server.path-resource-name-mapping`<br>
+: **Environment Variable**: `DD_TRACE_HTTP_SERVER_PATH_RESOURCE_NAME_MAPPING`<br>
+**Default**: `{}` (empty) <br>
+Maps HTTP request paths to custom resource names. Provide a comma‑separated list of `pattern:resource_name` pairs, where `pattern` is an [Ant‑style path pattern][20] that must match the value of the `http.path_group` span tag. Mappings are evaluated in order of priority, and the first matching rule applies. Unmatched request paths use the default normalization behavior.<br>
```

Comment:
You should add that using `*` as a value will leave the value unchanged. For instance, given this rule `/test/**:*`, when tracing a http get call for `/test/some/path`, the http server span will have `GET /test/some/path` as resource name